libs/langchain_v1/tests/unit_tests/test_pytest_config.py

```diff
@@ -6,6 +6,4 @@
 def test_socket_disabled() -> None:
     """This test should fail."""
     with pytest.raises(pytest_socket.SocketBlockedError):
-        # Ignore S113 since we don't need a timeout here as the request
-        # should fail immediately
-        requests.get("https://www.example.com")  # noqa: S113
+        requests.get("https://www.example.com", timeout=1)
```

Comment:
I think it's easier to set the timeout and check that the error is SocketBlockedError and not TimeoutError.